examples/07_advanced/plot_surface_bids_analysis.py

```diff
@@ -72,8 +68,6 @@
     task_label="languagelocalizer",
     space_label="",
     img_filters=[("desc", "preproc")],
-    sub_labels=["01", "02", "05", "08"],  # comment to run all subjects
```

Comment:
running on all subjects as discussed

TODO: check how much run time is increased

Reply:
[new build time](https://github.com/nilearn/nilearn/actions/runs/12653086833/job/35258014308?pr=4442#step:13:235):

- ../examples/07_advanced/plot_surface_bids_analysis.py:                      112.50 sec   1637.4 MB

[Build time with fewer subjects](https://github.com/nilearn/nilearn/actions/runs/12648293186/job/35248275649#step:13:25265) on main:

- ../examples/07_advanced/plot_surface_bids_analysis.py:                       74.87 sec    742.6 MB

Don't think that shaving 30 seconds on a build time of 90 minutes is worth worrying about for now, so let's run all subjects.